js/metadata.js

```diff
@@ -17,7 +17,7 @@ function initialize_metadata() {
   // TODO: Try more places in the MEI
   var composer, resp, resps = titleStmt.getElementsByTagName("respStmt");
   if(resps.length == 0){
-    let resp = mei.createElement("respStmt");
+    resp = mei.createElement("respStmt");
```

Comment:
Might `var resp` be better than `let resp` or plan `resp`? The latter implicitly creates a global property `window.resp`, I think.

Reply:
`resp` gets declared `var` along with `composer` and `resps` on line 18, so it's function-local at least.